bcdoc/restdoc.py

```diff
@@ -138,6 +138,10 @@ def path(self):
     def path(self, value):
         self._path = value
 
+    @property
+    def available_sections(self):
+        return list(self._structure.keys())
```

Comment:
Right, I'm saying you don't need the .keys() call.  Calling list on a dict will give you the keys.